src/main/java/com/solvd/laba/carina/homework/aptoide/objects/AppPageStaticElement.java

```diff
@@ -0,0 +1,30 @@
+package com.solvd.laba.carina.homework.aptoide.objects;
+
+public enum AppPageStaticElement implements StaticElement{
+    ANIMATED_COLOR_VIEW("cm.aptoide.pt:id/animated_color_view", ""),
+    VIEW_PAGE("cm.aptoide.pt:id/view_pager",""),
+    WIZARD_ICON("cm.aptoide.pt:id/wizard_icon",""),
+    MESSAGE_GROUP("cm.aptoide.pt:id/message_group",""),
+    MESSAGE_DISCOVER_BEST_APPS("cm.aptoide.pt:id/title","Discover the Best Apps"),
```

Comment:
An interesting solution, but... Enum is not a dynamic element, and in the application the text can change, it is better to create a locator on the page with the id of the element and store the text in the test itself or also on the page